pkg/gke/gke.go

```diff
@@ -0,0 +1,127 @@
+package gke
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/user"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func BuildCrbObject() (*unstructured.Unstructured, error) {
+	user, err := getActiveUser()
```

Comment:
the user should be a parameter, that way the function is more easily testable

Reply:
That function is quite strait-forward to just forming the clusterrolebinding with most fields are fixed. `user` is the only custom field, so I don't think we need unit-test for it. The `getActiveUser()` would need tests

Reply:
Well, we need to test this in two different ways:

 - Assume that the content of this CRB object can change and the software will still work. In this case we would need to cover this with integration tests.
 - Ensure that (only) the current definition works: that way you need to ensure that if someone changes it, he should know what he is doing and modify the unit test that cover this to the new format.

As far as I know we are not going to add integration tests for GKE (at least for the moment) so with a unit test at least you ensure that no ones modify it by accident. 

In any case, I am okay with not adding now a unit test now for this, but if in the future we discover a bug here and we want to add a unit test, it will be easier if the `getActiveUser` is outside this function.